bayesian/utils.py

```diff
@@ -64,16 +63,19 @@ def get_user_email(user_profile):
         return default_email
 
 
-def server_create_component_bookkeeping(ecosystem, name, version, user_profile):
+def create_component_bookkeeping(ecosystem, packages_list,
+                                 user_id, user_agent, manifest_hash, request_id):
     """Run the component analysis for given ecosystem+package+version."""
     args = {
-        'external_request_id': uuid.uuid4().hex,
         'data': {
```

Comment:
Why are the other fields not in the data dict?

Reply:
No particular reason.I ll move it.

Reply:
Resolved.